api/compose.yml

```diff
@@ -53,3 +53,12 @@ services:
       DJANGO_NGINX_UPSTREAM_URL: web:50280
     depends_on:
       - web
+
+  httpbin:
+    profiles:
+      - api
+    # Use go-httpbin because it's only 15 MB compared to upstream HTTPBin's 204 MB
+    # That helps reduce the network burden of spinning up a development environment
+    image: docker.io/mccutchen/go-httpbin:v2.14.0
+    expose:
+      - "8080"
```

Comment:
I tried a few approaches to testing the trace config, and this was the one that finally worked. Here's what I tried and why they didn't work:

1. I tried pook first, of course. However, the way pook hooks into aiohttp stops it from executing its trace config altogether, so it was useless for testing this.
2. I tried using pytest-httpbin, which runs an httpbin Python server during the pytest suite run, and provides some useful fixtures. However, httpbin itself is very badly out of date and incompatible with other dependencies of the API: https://github.com/postmanlabs/httpbin/issues/712
3. Finally, just running httpbin inside the compose stack, which is what I've done here.

I'm open to suggestions for other approaches that will work and avoid adding this dependency into the stack.